pg2kinesis/__main__.py

```diff
@@ -84,6 +110,9 @@ def __call__(self, change):
         progress_msg = 'xid: {:12} win_count:{:>10} win_size:{:>10}mb cum_count:{:>10} cum_size:{:>10}mb'
 
         for fmt_msg in fmt_msgs:
+            if not self.should_send_to_kinesis(fmt_msg):
+                fmt_msg = None
+
             did_put = self.writer.put_message(fmt_msg)
```

Comment:
Do we put_message with 'None' as the message?

Reply:
yes, i believe this was already being done to ensure that the replication slot is flushed. 

with filtering out operations, this is really important, because you could be filtering out most operations, which if we didn't ensure flushing on a regular basis, would lead to the replication slot growing unchecked. It must be periodically flushed to avoid eating up the entire free disk space.